core/App/ehsm_napi.cpp

```diff
@@ -1139,83 +1140,549 @@ char* NAPI_ExportDataKey(const char* cmk_base64,
 }
 
 /*
-@return
-[string] json string
+ * process receive msg2 json string to sgx_ra_msg2_t
+ *  @param ra_msg2 : receive msg2 json string
+ *  @param req_msg2 : return sgx_ra_msg2_t
+ *  @param msg2_size : return sgx_ra_msg2_t real size
+ */
+ehsm_status_t ra_proc_msg2(std::string ra_msg2, sgx_ra_msg2_t *req_msg2,uint32_t *msg2_size)
+{
```

Comment:
add static

Reply:
The function will be move to marshal.cpp